[masonry] Update css-grid parsing tests to grid-lanes

stwrt opened the pull request:
#### 95dcfedeacd765d5579e083a19cac8f51d135c89
<pre>
[masonry] Update css-grid parsing tests to grid-lanes
<a href="https://bugs.webkit.org/show_bug.cgi?id=302784">https://bugs.webkit.org/show_bug.cgi?id=302784</a>
<a href="https://rdar.apple.com/problem/165046164">rdar://problem/165046164</a>

Reviewed by Elika Etemad.

We had several grid tests that had not been synced from the upstream WPT. When I sync them I saw they were still using display: masonry.
Updated the tests to display: grid-lanes.

* LayoutTests/imported/w3c/web-platform-tests/css/css-grid/parsing/grid-template-columns-intrinsic-auto-repeat-computed-implicit-track.tentative-expected.txt
* LayoutTests/imported/w3c/web-platform-tests/css/css-grid/parsing/grid-template-columns-intrinsic-auto-repeat-computed-implicit-track.tentative.html
* LayoutTests/imported/w3c/web-platform-tests/css/css-grid/parsing/grid-template-columns-intrinsic-auto-repeat-computed-withcontent.tentative.html
* LayoutTests/imported/w3c/web-platform-tests/css/css-grid/parsing/grid-template-columns-intrinsic-auto-repeat-computed.tentative.html
* LayoutTests/imported/w3c/web-platform-tests/css/css-grid/parsing/grid-template-rows-intrinsic-auto-repeat-computed-implicit-track.tentative-expected.txt
* LayoutTests/imported/w3c/web-platform-tests/css/css-grid/parsing/grid-template-rows-intrinsic-auto-repeat-computed-implicit-track.tentative.html
* LayoutTests/imported/w3c/web-platform-tests/css/css-grid/parsing/grid-template-rows-intrinsic-auto-repeat-computed-withcontent.tentative.html
* LayoutTests/imported/w3c/web-platform-tests/css/css-grid/parsing/grid-template-rows-intrinsic-auto-repeat-computed.tentative.html

Canonical link: <a href="https://commits.webkit.org/303289@main">https://commits.webkit.org/303289@main</a>
</pre>
<!--EWS-Status-Bubble-Start-->
https://github.com/WebKit/WebKit/commit/86151d2553e157728d85040e4f238a3e5e5adb8d

| Misc | iOS, visionOS, tvOS & watchOS  | macOS  | Linux |  Windows | Apple Internal |
| ----- | ---------------------- | ------- |  ----- |  --------- | ------ |
| [✅ 🧪 style](https://ews-build.webkit.org/#/builders/38/builds/131860 "Passed style check") | [✅ 🛠 ios](https://ews-build.webkit.org/#/builders/159/builds/4352 "Built successfully") | [✅ 🛠 mac](https://ews-build.webkit.org/#/builders/138/builds/42869 "Built successfully") | [✅ 🛠 wpe](https://ews-build.webkit.org/#/builders/5/builds/139372 "Built successfully") | [✅ 🛠 win](https://ews-build.webkit.org/#/builders/59/builds/83745 "Built successfully") | [❌ 🛠 ios-apple](https://ews-bridge.webkit.apple.com/builds/sw/T-276/f2c6d2ed-bd8b-4d70-a499-64172ae81bd0/7217bf87-a7ba-4298-a40f-9a3a6777d1e7) 
| | [✅ 🛠 ios-sim](https://ews-build.webkit.org/#/builders/155/builds/4289 "Built successfully") | [✅ 🛠 mac-AS-debug](https://ews-build.webkit.org/#/builders/156/builds/4114 "Built successfully") | [✅ 🧪 wpe-wk2](https://ews-build.webkit.org/#/builders/34/builds/100789 "Passed tests") | [  ~~🧪 win-tests~~](https://ews-build.webkit.org/#/builders/60/builds/68196 "The change is no longer eligible for processing. Pull Request was already closed when EWS attempted to process it.") | [✅ 🛠 mac-apple](https://ews-bridge.webkit.apple.com/builds/sw/T-276/f2c6d2ed-bd8b-4d70-a499-64172ae81bd0/a52df3d4-c2a8-4500-9512-a26d6f869ad5) 
| [✅ 🧪 webkitperl](https://ews-build.webkit.org/#/builders/11/builds/134806 "Passed tests") | [  ~~🧪 ios-wk2~~](https://ews-build.webkit.org/#/builders/162/builds/3070 "The change is no longer eligible for processing. Pull Request was already closed when EWS attempted to process it.") | [✅ 🧪 api-mac](https://ews-build.webkit.org/#/builders/18/builds/118098 "Passed tests") | [✅ 🧪 api-wpe](https://ews-build.webkit.org/#/builders/41/builds/81579 "Passed tests") | | [✅ 🛠 vision-apple](https://ews-bridge.webkit.apple.com/builds/sw/T-276/f2c6d2ed-bd8b-4d70-a499-64172ae81bd0/e6e9aba8-6b10-439a-864c-58c2700fcdae) 
| | [✅ 🧪 ios-wk2-wpt](https://ews-build.webkit.org/#/builders/154/builds/2958 "Passed tests") | [  ~~🧪 api-mac-debug~~](https://ews-build.webkit.org/#/builders/165/builds/808 "The change is no longer eligible for processing. Commit was outdated when EWS attempted to process it.") | [✅ 🛠 wpe-cairo](https://ews-build.webkit.org/#/builders/65/builds/82592 "Built successfully") | | 
| | [✅ 🧪 api-ios](https://ews-build.webkit.org/#/builders/13/builds/111702 "Passed tests") | [✅ 🧪 mac-wk1](https://ews-build.webkit.org/#/builders/135/builds/36223 "Passed tests") | [✅ 🛠 gtk](https://ews-build.webkit.org/#/builders/2/builds/142015 "Built successfully") | | 
| | [✅ 🛠 vision](https://ews-build.webkit.org/#/builders/153/builds/4021 "Built successfully") | [✅ 🧪 mac-wk2](https://ews-build.webkit.org/#/builders/136/builds/36802 "Passed tests") | [✅ 🧪 gtk-wk2](https://ews-build.webkit.org/#/builders/1/builds/109164 "Passed tests") | | 
| | [✅ 🛠 vision-sim](https://ews-build.webkit.org/#/builders/160/builds/4102 "Built successfully") | [✅ 🧪 mac-AS-debug-wk2](https://ews-build.webkit.org/#/builders/161/builds/3520 "Passed tests") | [✅ 🧪 api-gtk](https://ews-build.webkit.org/#/builders/21/builds/109330 "Passed tests") | | 
| [✅ 🛠 🧪 merge](https://ews-build.webkit.org/#/builders/19/builds/27705 "Built successfully and passed tests") | [✅ 🧪 vision-wk2](https://ews-build.webkit.org/#/builders/164/builds/3060 "Passed tests") | [✅ 🧪 mac-wk2-stress](https://ews-build.webkit.org/#/builders/8/builds/114379 "Passed tests") | [✅ 🛠 playstation](https://ews-build.webkit.org/#/builders/134/builds/57236 "Built successfully") | | 
| | [✅ 🛠 tv](https://ews-build.webkit.org/#/builders/158/builds/4075 "Built successfully") | [✅ 🧪 mac-intel-wk2](https://ews-build.webkit.org/#/builders/137/builds/32779 "Passed tests") | | | 
| | [✅ 🛠 tv-sim](https://ews-build.webkit.org/#/builders/157/builds/3907 "Built successfully") | | | | 
| | [✅ 🛠 watch](https://ews-build.webkit.org/#/builders/163/builds/4167 "Built successfully") | | | | 
| | [✅ 🛠 watch-sim](https://ews-build.webkit.org/#/builders/152/builds/4035 "Built successfully") | | | | 
<!--EWS-Status-Bubble-End-->

fantasai reviewed (approved):
You need to fix the double `;;` but otherwise lgtm.

stwrt committed:
[masonry] Update css-grid parsing tests to grid-lanes
https://bugs.webkit.org/show_bug.cgi?id=302784
rdar://problem/165046164

Reviewed by Elika Etemad.

We had several grid tests that had not been synced from the upstream WPT. When I sync them I saw they were still using display: masonry.
Updated the tests to display: grid-lanes.

* LayoutTests/imported/w3c/web-platform-tests/css/css-grid/parsing/grid-template-columns-intrinsic-auto-repeat-computed-implicit-track.tentative-expected.txt
* LayoutTests/imported/w3c/web-platform-tests/css/css-grid/parsing/grid-template-columns-intrinsic-auto-repeat-computed-implicit-track.tentative.html
* LayoutTests/imported/w3c/web-platform-tests/css/css-grid/parsing/grid-template-columns-intrinsic-auto-repeat-computed-withcontent.tentative.html
* LayoutTests/imported/w3c/web-platform-tests/css/css-grid/parsing/grid-template-columns-intrinsic-auto-repeat-computed.tentative.html
* LayoutTests/imported/w3c/web-platform-tests/css/css-grid/parsing/grid-template-rows-intrinsic-auto-repeat-computed-implicit-track.tentative-expected.txt
* LayoutTests/imported/w3c/web-platform-tests/css/css-grid/parsing/grid-template-rows-intrinsic-auto-repeat-computed-implicit-track.tentative.html
* LayoutTests/imported/w3c/web-platform-tests/css/css-grid/parsing/grid-template-rows-intrinsic-auto-repeat-computed-withcontent.tentative.html
* LayoutTests/imported/w3c/web-platform-tests/css/css-grid/parsing/grid-template-rows-intrinsic-auto-repeat-computed.tentative.html

Canonical link: https://commits.webkit.org/303289@main
